client/src/deploy/creation_common.rs

```diff
@@ -345,6 +359,27 @@ mod arg_simple {
             ("account_hash", AccountHash::cl_type()),
             ("uref", CLType::URef),
             ("public_key", CLType::PublicKey),
+            ("opt_bool", CLType::Option(Box::new(CLType::Bool))),
+            ("opt_i32", CLType::Option(Box::new(CLType::I32))),
+            ("opt_i64", CLType::Option(Box::new(CLType::I64))),
+            ("opt_u8", CLType::Option(Box::new(CLType::U8))),
+            ("opt_u32", CLType::Option(Box::new(CLType::U32))),
+            ("opt_u64", CLType::Option(Box::new(CLType::U64))),
+            ("opt_u128", CLType::Option(Box::new(CLType::U128))),
+            ("opt_u256", CLType::Option(Box::new(CLType::U256))),
+            ("opt_u512", CLType::Option(Box::new(CLType::U512))),
+            ("opt_unit", CLType::Option(Box::new(CLType::Unit))),
+            ("opt_string", CLType::Option(Box::new(CLType::String))),
+            ("opt_key", CLType::Option(Box::new(CLType::Key))),
```

Comment:
I'm not seeing the opt args coming through the help:
`./target/release/casper-client put-deploy --help`
```    -a, --session-arg <NAME:TYPE='VALUE'>...
            For simple CLTypes, a named and typed arg which is passed to the Wasm code. To see an example for each type,
            run '--show-arg-examples'. This arg can be repeated to pass multiple named, typed args, but can only be used
            for the following types: bool, i32, i64, u8, u32, u64, u128, u256, u512, unit, string, key, account_hash,
            uref, public_key
```
This looks like they should be coming out on Line 394.

Reply:
Is this maybe an old version of the client? (Did you build a debug version?)

Locally I see:
```
        --payment-arg <NAME:TYPE='VALUE' OR NAME:TYPE=null>...
            For simple CLTypes, a named and typed arg which is passed to the Wasm code. To see an example for each type,
            run '--show-arg-examples'. This arg can be repeated to pass multiple named, typed args, but can only be used
            for the following types: bool, i32, i64, u8, u32, u64, u128, u256, u512, unit, string, key, account_hash,
            uref, public_key, opt_bool, opt_i32, opt_i64, opt_u8, opt_u32, opt_u64, opt_u128, opt_u256, opt_u512,
            opt_unit, opt_string, opt_key, opt_account_hash, opt_uref, opt_public_key
```